ogc/src/main/scala/geotrellis/server/ogc/wcs/WcsModel.scala

```diff
@@ -24,14 +24,16 @@ case class WcsModel(
   sources: OgcSourceRepository
 ) {
 
-  def getLayers(p: GetCoverageWcsParams): List[OgcLayer] =
-    sources
-      .find(p.toQuery)
-      .map {
+  def getLayers(p: GetCoverageWcsParams): List[OgcLayer] = {
+    val logger = org.log4s.getLogger
```

Comment:
logger is usually created per class instance, not per a function call; It's better to put it into WcsModel class private field.